casaos-fix-docker-api-version/test-script.sh

```
@@ -1592,6 +1594,143 @@ TESTEOF
+exit 0
+EOF
+  chmod +x "$mock_dir/dpkg"
+  
+  # Save PATH
+  local old_path="$PATH"
+  export PATH="$mock_dir:$PATH"
+  
+  print_info "Testing: timeout 10 dpkg -l"
+  
+  # Test the timeout command directly
+  local start_time=$(date +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)